app/utils/whitelabel.ts

```diff
@@ -45,7 +45,6 @@ interface WhiteLabelSite {
   intercom: boolean;
   logoLinkDestination: string;
   navLogoStyle: string;
-  refinementListLimit: number;
```

Comment:
This config value is not currently need and arguably never was. Algolia's default is 10 and in cases where we are filtering by either a static list of refinements or one returned by the API (in the case of the subcategories endpoint), we need a high value to ensure that all desired categories are returned to begin with